pegasus/sites.v3/hourofcode.com/public/js/signup-form.js

```diff
@@ -44,23 +44,23 @@ $(document).ready(function () {
       $('#hoc-entire-school').show();
       // continue button goes to census questions on click
       $('.continue-btn').show();
-      $('#submit').hide();
+      $('#submit-button').hide();
     } else if (($('#hoc-event-type').val() === 'in_school')){
     // in-school & NOT US
       $('#school-name-field').show();
       $('#organization-name-field').hide();
       $('#hoc-event-location-field').show();
       $('#hoc-entire-school').show();
       $('.continue-btn').hide();
-      $('#submit').show();
+      $('#submit-button').show();
     } else if ($('#hoc-event-type').val() === 'out_of_school') {
       // out of school, either US or non-US
       $('#organization-name-field').show();
       $('#hoc-event-location-field').show();
       $('#school-name-field').hide();
       $('#hoc-entire-school').hide();
       $('.continue-btn').hide();
-      $('#submit').show();
+      $('#submit-button').show();
```

Comment:
I would expect to see #continue-btn and #submit-btn or #continue-button and #submit-button but not .continue-btn and #submit-button. Can we make them more consistent?

Reply:
Done.